RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -38,3 +38,5 @@
                             ConvStep*
                             conversionStepTracks
                             )
+_iterTracking_Phase2PU140 = iterTracking.copyAndExclude([InitialStepPreSplitting])
+eras.trackingPhase2PU140.toReplaceWith(iterTracking, _iterTracking_Phase2PU140)
```

Comment:
On a related note, you need to add customizations (at least) to
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Configuration/StandardSequences/python/Reconstruction_cff.py#L75
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/RecoPixelVertexing/Configuration/python/RecoPixelVertexing_cff.py
(see #13666 for phase1)